bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/ChooseWorkspaceDialog.java

```diff
@@ -452,9 +451,7 @@ private void createWorkspaceBrowseRow(Composite parent) {
 		Button browseButton = new Button(panel, SWT.PUSH);
 		browseButton.setText(IDEWorkbenchMessages.ChooseWorkspaceDialog_browseLabel);
```

Comment:
"Workspace:" as tooltip looks poor, please adjust the text to be real tooltip

Reply:
Feel free to propose a better text ;-)

Reply:
"Enter a directory as workspace" would be better.

Reply:
my 2 cents:
A path combobox and an attached `...` button are standard UI elements, so I doubt if the button requires a label at all.
Though, if this is considered a bugfix / UI polish, there shouldn't be significant UI (text) changes IINM, and making the dialog elements responsive is sufficient for now, no?

Reply:
I removed the useless text from the tooltip now.